Goal: During the changes on libmisc/setupenv.c, a piece of code was moved
without its comment. This comment is now at the wrong place.

Status wrt to upstream: 

Index: shadow-4.0.12/libmisc/setupenv.c
===================================================================
--- shadow-4.0.12.orig/libmisc/setupenv.c	2005-08-17 11:49:12.000000000 +0200
+++ shadow-4.0.12/libmisc/setupenv.c	2005-08-17 11:51:18.000000000 +0200
@@ -234,9 +234,6 @@
 
 	addenv ("SHELL", info->pw_shell);
 
-	/*
-	 * Create the PATH environmental variable and export it.
-	 */
 
 	/*
 	 * Export the user name.  For BSD derived systems, it's "USER", for
@@ -247,6 +244,9 @@
 	addenv ("LOGNAME", info->pw_name);
 
 #ifndef USE_PAM
+	/*
+	 * Create the PATH environmental variable and export it.
+	 */
 	cp = getdef_str ((info->pw_uid == 0) ? "ENV_SUPATH" : "ENV_PATH");
 
 	if (!cp) {
